src/libssh2.mk

```diff
@@ -21,7 +21,8 @@ define $(PKG)_BUILD
     cd '$(1)' && ./configure \
         $(MXE_CONFIGURE_OPTS) \
         --disable-examples-build \
-        --with-crypto=libgcrypt \
+        --disable-rpath \
+        --with-crypto=wincng \
         LIBS="`$(PREFIX)/$(TARGET)/bin/libgcrypt-config --libs`" \
```

Comment:
This `libgcrypt-config` line should no longer be required.

Reply:
Thanks, I'll remove it.